.travis.yml

```
@@ -144,5 +144,5 @@ env:
 # Send cron job notification to our Slack room.
 notifications:
   slack:
-    on_pull_requests: false
-    secure: Wxsh9jI4Noq8BQtEl7CjfsHtQRmVo5l7jCDkY0W7zqHqnPRUBiAd847uPcBRUcWE8zYWDLXqtTHJMc/ctyLodentArgvHTtBIw6gp2QaFFFYe1WE0tPSYmhGa9GjI46D2/a4rCjNo1uIpytoHGTSp6RNBhY3TFKqZf+qy6RnIcU=
+    rooms:
+      - secure: Wxsh9jI4Noq8BQtEl7CjfsHtQRmVo5l7jCDkY0W7zqHqnPRUBiAd847uPcBRUcWE8zYWDLXqtTHJMc/ctyLodentArgvHTtBIw6gp2QaFFFYe1WE0tPSYmhGa9GjI46D2/a4rCjNo1uIpytoHGTSp6RNBhY3TFKqZf+qy6RnIcU= #dev-ci
```

The examples on [Configuring Build Notifications](https://docs.travis-ci.com/user/notifications/#configuring-slack-notifications) don't have a space between the token and the room. Should it be removed?

```suggestion
      - secure: Wxsh9jI4Noq8BQtEl7CjfsHtQRmVo5l7jCDkY0W7zqHqnPRUBiAd847uPcBRUcWE8zYWDLXqtTHJMc/ctyLodentArgvHTtBIw6gp2QaFFFYe1WE0tPSYmhGa9GjI46D2/a4rCjNo1uIpytoHGTSp6RNBhY3TFKqZf+qy6RnIcU=#dev-ci
```